tests/release/service/happy_path.go

```diff
@@ -75,7 +75,7 @@ var _ = framework.ReleaseServiceSuiteDescribe("Release service happy path", Labe
 		_, err = fw.AsKubeAdmin.HasController.CreateApplication(releasecommon.ApplicationNameDefault, devNamespace)
 		Expect(err).NotTo(HaveOccurred())
 
-		component = releasecommon.CreateComponent(*fw, devNamespace, releasecommon.ApplicationNameDefault, releasecommon.ComponentName, releasecommon.GitSourceComponentUrl, "", ".", "Dockerfile", constants.DefaultDockerBuildPipelineBundle)
+		component, baseBranchName, pacBranchName = releasecommon.CreateComponent(*fw, devNamespace, releasecommon.ApplicationNameDefault, releasecommon.ComponentName, releasecommon.DcMetroMapGitSourceURL, releasecommon.DcMetroMapGitRevision, ".", "Dockerfile", constants.DefaultDockerBuildPipelineBundle)
 
 		_, err = fw.AsKubeAdmin.ReleaseController.CreateReleasePlan(releasecommon.SourceReleasePlanName, devNamespace, releasecommon.ApplicationNameDefault, managedNamespace, "", nil, nil)
```

Comment:
Please help to move  line#80 & 81 ( related to CreateReleasePlan() ) before Line#78  (CreateComponent() ). Thanks

Reply:
Updated.

Reply:
@jinqi7 this change does not help I guess. [This step](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/konflux-ci_e2e-tests/1273/pull-ci-konflux-ci-e2e-tests-main-redhat-appstudio-e2e/1819003271407210496#1:build-log.txt%3A1661) is still failing. Could you please take a look.